src/library_dylink.js

```diff
@@ -1166,7 +1182,10 @@ var LibraryDylink = {
 #endif
     result = lib.module[symbol];
 
-    if (typeof result == 'function') {
+    // Workaround for closure compiler error if f.stub is accessed directly
+    // on the line below.
```

Comment:
Weird error... what's the error message, if you still have it?

Reply:
Closure says that `Function` does not have a `stub` attribute.    Which is true for most functions, but we sometimes at them, which I guess is not something closure likes in general (attributes that might or might not exist).  Should I add more detail to the comment?

Reply:
No need, thanks, I was just curious.

Reply:
I think the problem is that closure treat the `if (typeof result == 'function')` type refinement and will then assume that `result` to be of type `Function`.